src/main/java/org/mate/endpoints/LegacyEndpoint.java

```diff
@@ -124,6 +124,9 @@ private String handleRequest(String cmdStr) {
         if (cmdStr.startsWith("contrastratio"))
             return ImageHandler.calculateConstratRatio(cmdStr);
 
+        if (cmdStr.startsWith("surroundingColor"))
```

Comment:
Should not be added to the LegacyEndpoint. Instead a new Endpoint for things related to accessibility should be added to the router.